### ISSUE TITLE
Performance test

### ISSUE THREAD

**Issue by wangsun1996** [liming-ai/AlignDet#14]
After I finish pre-training and fine-tuning, how do I get the performance metrics of the model?

**Comment by liming-ai** [comment]
> After I finish pre-training and fine-tuning, how do I get the performance metrics of the model?

Hi, in the normal training process, the final performance metric (mAP) should be printed automatically. If you want to test a model, please refer to [the test script](https://github.com/liming-ai/AlignDet/blob/master/tools/dist_test.sh).

For more information about the use of mmdet, please refer to [mmdet official tutorial](https://mmdetection.readthedocs.io/en/stable/).
